specification/web/resource-manager/readme.md

```diff
@@ -464,3 +464,20 @@ java:
 regenerate-manager: true
 generate-interface: true
 ```
+
+### Tag: package-2016-09

```

Comment:
There's already such a tag at https://github.com/Azure/azure-rest-api-specs/pull/4034/files#diff-abbc8817428c1cce3cd52fe52a4f1b75R71

Reply:
Will take a look

Reply:
I named the tag incorrectly here